SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -189,6 +189,22 @@ UnitTest {
 		}
 	}
 
+	testForException { |func, errorClass|
+		var test = if(errorClass.isNil) { _.notNil } { _.isKindOf(errorClass.asClass) };
```

Comment:
I really do believe this argument should be mandatory (i.e. no "default behavior" for nil). There's no benefit to being lenient in this case, it only leaves space for user error. Besides, it is also possible to get this kind of all-pass behavior by specifying `Exception` or `Object` as the error type.